netjsonconfig/backends/openwrt/converters/zerotier.py

```diff
@@ -0,0 +1,27 @@
+from ...zerotier.converters import ZeroTier as BaseZeroTier
+from ..schema import schema
+from .base import OpenWrtConverter
+
+
+class ZeroTier(OpenWrtConverter, BaseZeroTier):
+    _uci_types = ['zerotier']
+    _schema = schema['properties']['zerotier']['items']
+
+    def __intermediate_vpn(self, vpn):
+        vpn.update(
+            {
+                '.name': self._get_uci_name(vpn.pop('name')),
+                '.type': 'zerotier',
+                'join': [vpn.pop('id')],
+                'enabled': not vpn.pop('disabled', False),
+            }
+        )
+        return super().__intermediate_vpn(vpn, remove=[''])
+
+    def __netjson_vpn(self, vpn):
+        vpn['id'] = str(vpn.pop('join')[0])
```

Comment:
Can you point out where the "join" property is defined in the schema? I need to verify wether the schema enforces having at least 1 list member otherwise this code can break.

Reply:
Sure, I'm utilizing the [`id` array property](https://github.com/openwisp/netjsonconfig/pull/274/files#diff-cbba1cabcdaea2e99f955c45dfdde7328a4b83821a1d5780f8d0a22b66eb6bd1R951-R963) of Zerotier with a `minItem` value of 1.